### PR TITLE
Librarian boss and Emon update

### PR DESCRIPTION
**Pull request by Miidgiit**
Emondeus= Demon now

Soul Sucker has penetration= goal is to always hit.. always drain your life..but for very little.. But little does add up.

Added the Librarian boss to -1  its a skeleton with a flail. I've heard talks of it being from OG so I thought I'd pop it out there.

most -1 mobs are 8 hp. It's 40+ hp with 1 skill higher chance to hit. 1 hrish respawn.

### CODE REVIEW

File: Segments/Kesmai.mapproj [hunk 1]
```diff
@@ -145233,18 +145233,18 @@ else
         
         IsTethered = true,
     };
-    dragon.AddStatus(new NightVisionStatus(dragon));
+    demon.AddStatus(new NightVisionStatus(dragon));
```
Comment (Chilly021) [inline Segments/Kesmai.mapproj:145236]: You need to change the new nvstatus(dragon) to new nvstatus(demon), otherwise compiler issues.

**Review by Chilly021 — CHANGES_REQUESTED** [review]
One comment. The constructor needs to pass in the demon param. I'll be back tonight if you need anything else.

**Review by Chilly021 — COMMENTED** [review]
Does the librarian need a unique body type? Or is a default skeleton all you wanted?

**Comment by Miidgiit** [comment]
simple skeleton for now.

**Comment by Miidgiit** [comment]
i changed his color slightly. (end of my changes)

**Review by Chilly021 — APPROVED** [review]

**Review by Leolus — APPROVED** [review]

**Comment by sproatba** [comment]
Looking at this on my phone so I don't have the editor to help, but it looks like a kobold with all zeros was added.  Not sure what that will do.

**Comment by Miidgiit** [comment]
image

So yeah. I have no clue how to actually delete a made region.. I dont see an option anywhere so I just turned it to zero.

**Review by zrikzlok — APPROVED** [review]

**Pull request merged**